### PR TITLE
Fix cemu, citra, ppsspp, rpcs3, xemu and more

### LINKED ISSUE #631: Fix autoupdate for Xenia Canary
Xenia Canary is not autoupdating at the moment (all other emulators and games happens same).

### PR DESCRIPTION
**Pull request by beyondmeat**
## What package release type is this?

> - [x] stable
> - [ ] beta
> - [ ] dev/nightly/canary

closes #631

### CODE REVIEW

**Comment by beyondmeat** [comment]
@Calinou

File: bucket/ppsspp.json [hunk 1]
```diff
@@ -7,9 +7,13 @@
...
     },
     "url": "https://ppsspp.org/files/1_12_3/ppsspp_win.zip",
-    "hash": "c29b1faf56df9c9926cfb77f874484571fbe8e59689b1a39091425ab309e6463",
-    "architecture": {
-        "64bit": {
+    "hash": "f909ae76b63984cd88f8c3e8ff56fd6e6bd7de98dd1ee441b5290ddce3fe7d94",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\")) {",
+        "   New-item \"$persist_dir\" -ItemType Directory | Out-Null",
+        "   New-item \"$persist_dir\\memstick\" -ItemType Directory | Out-Null",
```
Comment (Calinou) [inline bucket/ppsspp.json:14]:
```suggestion
        "   New-Item \"$persist_dir\" -ItemType Directory | Out-Null",
        "   New-Item \"$persist_dir\\memstick\" -ItemType Directory | Out-Null",
```

File: bucket/xenia-canary.json [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-    "architecture": {
-        "64bit": {
-            "url": "https://github.com/xenia-canary/xenia-canary/releases/download/experimental/xenia_canary.zip"
-        }
-    },
+    "url": "https://github.com/xenia-canary/xenia-canary/releases/download/6e753c6/xenia_canary.zip",
+    "hash": "b7dd6fea7b3237f17604e451d074e87ca77b4360867d10d09f02480376eae625",
     "pre_install": [
         "if (!(Test-Path \"$persist_dir\")) {",
         "   New-item \"$persist_dir\" -ItemType Directory | Out-Null",
```
Comment (Calinou) [inline bucket/xenia-canary.json:13]:
```suggestion
        "   New-Item \"$persist_dir\" -ItemType Directory | Out-Null",
```

File: bucket/pcsx2-dev.json [hunk 3]
```diff
@@ -14,24 +14,8 @@
...
-                    "PCSX2 (development)"
-                ]
-            ]
-        }
-    },
+    "url": "https://github.com/PCSX2/pcsx2/releases/download/v1.7.3058/pcsx2-v1.7.3058-windows-64bit-AVX2-Qt.7z",
+    "hash": "856366609310676f9088ef57e5d9b7961d1e0074696b2c59c975ee9dbd314325",
     "pre_install": [
         "if (!(Test-Path \"$persist_dir\")) {",
         "   New-item \"$persist_dir\" -ItemType Directory | Out-Null",
```
Comment (Calinou) [inline bucket/pcsx2-dev.json:21]:
```suggestion
        "   New-Item \"$persist_dir\" -ItemType Directory | Out-Null",
```

**Review by Calinou — APPROVED** [review]
Changes look good to me. Will merge after a rebase.

**Comment by Calinou** [comment]
Sorry, I was away while you rebased it. There are still two conflicts to fix.

**Comment by beyondmeat** [comment]
@Calinou done

**Pull request merged**